modules/gapi/include/opencv2/gapi/gmat.hpp

```diff
@@ -77,6 +77,18 @@ class GAPI_EXPORTS_W_SIMPLE GMat
      */
     GAPI_WRAP GMat();                       // Empty constructor
 
+    /**
+     * @brief Constructs a value-initialized GMat
+     *
+     * GMat may be associated with a buffer at graph construction time.
+     * It is useful when some operation has a Mat input which doesn't
+     * change during the program execution, and is set only once.
+     * In this case, there's no need to declare such GMat as graph input.
+     *
+     * @param m a cv::Mat buffer to associate with this GMat object.
+     */
+    GAPI_WRAP explicit GMat(cv::Mat m);     // Value-initialization constructor
```

Comment:
Wondering if it's really working from `python`, do you plan to add any test to check that, or it will be done later on?

Reply:
I believe it should work in the same way as for `GScalar(cv::Scalar)`, isn't it?